### PR TITLE
Reffer views to the View Code Style

### PR DESCRIPTION
**Pull request by CyanoFresh**
Reffer views to the View Code Style - https://github.com/yiisoft/yii2/blob/master/docs/internals/view-code-style.md

### CODE REVIEW

File: views/layouts/main.php [hunk 1]
```diff
@@ -21,47 +22,50 @@
...
-                    ['label' => 'Home', 'url' => ['/site/index']],
-                    ['label' => 'About', 'url' => ['/site/about']],
-                    ['label' => 'Contact', 'url' => ['/site/contact']],
-                    Yii::$app->user->isGuest ?
-                        ['label' => 'Login', 'url' => ['/site/login']] :
-                        ['label' => 'Logout (' . Yii::$app->user->identity->username . ')',
-                            'url' => ['/site/logout'],
-                            'linkOptions' => ['data-method' => 'post']],
+
+<div class="wrap">
```
Comment (cebe) [inline views/layouts/main.php]: why did you change indentation here, it is all part of body so indentation was just fine...

**Comment by cebe** [comment]
the rest looks good to me.

File: views/layouts/main.php (hunk 1 above)
Comment (CyanoFresh) [inline views/layouts/main.php]: So if the body has indentation, maybe beginPage() should have, too?

Comment (cebe) [inline views/layouts/main.php]: well, that is maybe too much. I think we can keep it this way.

File: views/site/contact.php [hunk 2]
```diff
@@ -15,43 +16,54 @@
...
+            Thank you for contacting us. We will respond to you as soon as possible.
+        </div>
+
+        <p>
+            Note that if you turn on the Yii debugger, you should be able
+            to view the mail message on the mail panel of the debugger.
+            <?php if (Yii::$app->mailer->useFileTransport): ?>
+                Because the application is in development mode, the email is not sent but saved as
+                a file under <code><?= Yii::getAlias(Yii::$app->mailer->fileTransportPath) ?></code>.
+                                                                                                    Please configure the
```
Comment (cebe) [inline views/site/contact.php]: here is an indentation issue.

File: views/layouts/main.php (hunk 1 above)
Comment (CyanoFresh) [inline views/layouts/main.php]: Thanks for answering. I will add indent. Can you help me? I've not understood how the pull requests works: now I should close pull request then make changes in my fork and then create new pull request?

Comment (CyanoFresh) [inline views/layouts/main.php]: Or there is another way?

Comment (cebe) [inline views/layouts/main.php]: no need to change anything here. just change the bug in contact.php I posted below.
to add changes to this PR, you just add another commit to the same branch and push it.

File: views/site/contact.php (hunk 2 above)
Comment (CyanoFresh) [inline views/site/contact.php]: Identetion from if/else statement. From View Code Style:
```
<!-- Separate PHP blocks are preferred for foreach, for, if etc. -->
<?php foreach ($posts as $post): ?>
    <!-- Note indentation level here. -->
```

Comment (cebe) [inline views/site/contact.php]: no, I mean the line 29 that contains "Please configure the" right above my commit, scroll a bit to the right to see it ;)

Comment (CyanoFresh) [inline views/site/contact.php]: Yeah. It's a bug in the new PhpStorm 9. Will report them.

**Comment by cebe** [comment]
Thank you!
